config/lib/helpers/cache.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
     name: 'broadcasts',
     ttl: parseInt(process.env.GAMBIT_BROADCASTS_CACHE_TTL, 10) || 3600,
   },
+  campaignConfigs: {
+    name: 'campaignConfigs',
+    ttl: parseInt(process.env.GAMBIT_CAMPAIGN_CONFIGS_CACHE_TTL, 10) || 3600,
```

Comment:
Minor. Since we have the same default `3600`, maybe worth moving to a single variable.